lib/ExpressRedisCache/route.js

```diff
@@ -166,46 +175,39 @@ module.exports = (function () {
           /** if it's cached, display cache **/
 
           if ( cache.length ) {
-            res.contentType(cache[0].type || "text/html");
-            if(binary){ //Convert back to binary buffer
-              res.send(new Buffer(cache[0].body, 'base64'));
-            }else{
-              res.send(cache[0].body);
-            }
+            res.socket.write(cache[0].body);
+            res.end();
           }
 
           /** otherwise, cache request **/
           else {
+            var _write = res.socket.write.bind(res.socket);
 
-            /** wrap res.send **/
-            var send = res.send.bind(res);
-
-            res.send = function (body) {
-
+            res.socket.write = function redisCacheWrite (body) {
               /** send output to HTTP client **/
-              var ret = send(body);
+              _write.apply(_write, Array.prototype.slice.call(arguments));
 
               /** convert binary to base64 string **/
               if(binary && typeof body !== 'string'){
```

Comment:
Also should not be required now since _write_ will (I think) always be called with a String